iphone/Classes/DeMarcelpociotCollectionviewCollectionView.h

```diff
@@ -8,7 +8,6 @@
 #import "TiUIView.h"
 #import "DeMarcelpociotCollectionviewCollectionViewProxy.h"
 #import "CHTCollectionViewWaterfallLayout.h"
-#import "M13ContextMenu.h"
```

Comment:
Will keep it as a fork for now then. Having it tied into the module is really no good idea and the used library is not even curated anymore.